consensus/clique/snapshot.go

```diff
@@ -126,11 +126,16 @@ func (s *Snapshot) copy() *Snapshot {
 	return cpy
 }
 
+// checkVote returns if it's a valid vote or not
+func (s *Snapshot) checkVote(address common.Address, authorize bool) bool {
```

Comment:
Please name this `validVote` (check doesn't really reveal what it checks). Also please update the method doc to something along the lines of.

```
// validVote returns whether it makes sense to cast the specified vote in the
// given snapshot context (e.g. don't try to add an already authorized signer).
```